src/components/button/button.css

```diff
@@ -0,0 +1,67 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.interactivity {
+  cursor: pointer;
+}
+
+.typography {
+  font-family: var(--mavenlink-type-font-family);
+  font-size: var(--mavenlink-type-button-font-size);
+  line-height: var(--spacing-x-large);
+}
+
+.button {
+  composes: typography;
+  composes: interactivity;
+  border-width: 1px;
+  border-radius: 3px;
+  border-style: solid;
+  padding-bottom: 0;
+  padding-left: var(--spacing-x-large);
+  padding-right: var(--spacing-x-large);
+  padding-top: 0;
+}
+
+.button:hover {
+  opacity: .7;
+}
+
+.button[disabled] {
+  background: #eee;
```

Comment:
Questions for design: are we okay with bypassing our palette?

Reply:
This is leftover from unicorn-ui (I pretty sure). I agree, let's replace this with something more from Mavenlink. We have an #ededed; which should be virtually the same visually...just use whatever that palette is. I don't think design needs to corroborate but might not be bad idea